docs/design-documents/schema_service.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
```

Comment:
Do we need this file? It's already embedded in the design document itself.

Reply:
Definitely not!